### PR TITLE
docs: google universal connector

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

docs: google universal connector

## Linear Issue Reference

* LOG-2620

## Testing

N/A

### CODE REVIEW

File: packages/connector-google/README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
-### Google Social Connector README
-placeholder
+### Google
+The Google connector provides a succinct way to let your application can use Google’s OAuth 2.0 authentication system for user login.
+
+# Prerequisite
+
+- A Google account
+- OAuth 2.0 credentials ( a client id an a client secret)
```
Comment (gao-sun) [inline packages/connector-google/README.md:7]:
```suggestion
- OAuth 2.0 credentials (client ID and client secret)
```

File: packages/connector-google/README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+
+You can create Web OAuth 2.0 credentials according to [the officer guide](https://developers.google.com/identity/protocols/oauth2/openid-connect#appsetup) and remember to create a client id for the web application.
```
Comment (gao-sun) [inline packages/connector-google/README.md:9]: what is officer?

**Review by gao-sun — CHANGES_REQUESTED** [review]
double check typos and align with another eng before proceeding

File: packages/connector-google/README.md [hunk 3]
```diff
@@ -1,2 +1,23 @@
-### Google Social Connector README
-placeholder
+### Google
+The Google connector provides a succinct way to let your application can use Google’s OAuth 2.0 authentication system for user login.
```
Comment (demonzoo) [inline packages/connector-google/README.md:2]: let your application ~~can~~ use

File: packages/connector-google/README.md (hunk 2 above)
Comment (demonzoo) [inline packages/connector-google/README.md:9]: according to => based on

File: packages/connector-google/README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 ### Google
-The Google connector provides a succinct way to let your application can use Google’s OAuth 2.0 authentication system for user login.
+The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system for users to login.
 
 # Prerequisite
 
 - A Google account
-- OAuth 2.0 credentials ( a client id an a client secret)
+- OAuth 2.0 credentials (client ID an a client secret)
```
Comment (wangsijie) [inline packages/connector-google/README.md:7]: and

Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated.

File: packages/connector-google/README.md (hunk 2 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated.

File: packages/connector-google/README.md (hunk 3 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated.

File: packages/connector-google/README.md (hunk 2 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated.

File: packages/connector-google/README.md [hunk 5]
```diff
@@ -1,2 +1,23 @@
...
+
+You follow the [Google Identity OAuth 2.0 Guideline](https://developers.google.com/identity/protocols/oauth2/openid-connect#appsetup) to create Web OAuth 2.0 credentials and don't forget to create a client ID for your web application. 
+Finally, you should get a client ID and client secret.
+
+# Setting up the client
+
+1. Select the **Credentials** page in API Console
+2. Click the client you have just created in the **OAuth 2.0 Client IDs** section
+3. Set up the ****Authorized JavaScript origins****, which are origins where the logto authorization page is hosted. In our case this will be `${your_logto_origin}`. eg.`https://logto.dev`.
+4. Set up the ****Authorized redirect URIs****, which is used to redirect the user to the application after the user has logged in. In our case, this will be `${your_logto_origin}/callback/google-universal`. eg. `https://logto.dev/callback/google-universal`.
```
Comment (IceHe) [inline packages/connector-google/README.md:17]: eg. should be e.g.

File: packages/connector-google/README.md [hunk 6]
```diff
@@ -1,2 +1,23 @@
...
+
+# Settings
+| Name | Type |
```
Comment (IceHe) [inline packages/connector-google/README.md:20]: better add a blank line between heading and content.

**Review by IceHe — COMMENTED** [review]
just curious why the doc title (1st line) is heading 3, and the rest section titles are heading 1?

File: packages/connector-google/README.md (hunk 5 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated

File: packages/connector-google/README.md [hunk 7]
```diff
@@ -1,2 +1,23 @@
-### Google Social Connector README
-placeholder
+### Google
+The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system for users to login.
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated

File: packages/connector-google/README.md (hunk 6 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated

File: packages/connector-google/README.md [hunk 8]
```diff
@@ -1,2 +1,25 @@
...
+
+The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system for users to login.
+
+## Prerequisite
+
+- A Google account
+- OAuth 2.0 credentials (client ID and client secret)
+
+You can follow the [Google Identity OAuth 2.0 Guideline](https://developers.google.com/identity/protocols/oauth2/openid-connect#appsetup) to create Web OAuth 2.0 credentials and don't forget to create a client ID for your web application. 
+Finally, you should get a client ID and client secret.
```
Comment (wangsijie) [inline packages/connector-google/README.md:11]: will?

File: packages/connector-google/README.md [hunk 9]
```diff
@@ -1,2 +1,25 @@
-### Google Social Connector README
-placeholder
+# Google
+
+The Google connector provides a succinct way for your application to use Google’s OAuth 2.0 authentication system for users to login.
```
Comment (demonzoo) [inline packages/connector-google/README.md:3]: I think `for users to login` is redundant and should be omitted

File: packages/connector-google/README.md (hunk 8 above)
Comment (demonzoo) [reply packages/connector-google/README.md:11]: nit: remove `a` or change it to `the`

**Review by demonzoo — APPROVED** [review]

File: packages/connector-google/README.md (hunk 9 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated.

File: packages/connector-google/README.md (hunk 8 above)
Comment (xiaoyijun) [reply packages/connector-google/README.md]: updated.

**Review by wangsijie — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

**Review by gao-sun — APPROVED** [review]

**Pull request merged**